km/km_snapshot.c

```diff
@@ -158,11 +158,7 @@ static inline void km_ss_recover_memory(int fd, km_gva_t tbrk_gva, km_payload_t*
                               fd,
                               phdr->p_offset);
                if (m == MAP_FAILED) {
-                  km_err(errno,
-                         "snapshot mmap[%d]: vaddr=0x%lx offset=0x%lx",
-                         i,
-                         phdr->p_vaddr,
-                         phdr->p_offset);
+                  km_err(2, "snapshot mmap[%d]: vaddr=0x%lx offset=0x%lx", i, phdr->p_vaddr, phdr->p_offset);
```

Comment:
Is there a reason you did not use km_err_e() here?
Same question for the change below.

Reply:
km_err() does print the errno.  Not thinking again.